### PR TITLE
Add currencySize prop

### PR DESCRIPTION
**Pull request by JacobBlomgren**

### CODE REVIEW

File: src/Atoms/Number/Number.tsx [hunk 1]
```diff
@@ -100,12 +102,25 @@ const NumberComponent: NumberComponentType = ({
...
+  );
+  if (typeof currencySize === 'undefined') return number;
+  return (
+    <>
+      <VisuallyHidden>
+        {number}
+        {` ${currency}`}
+      </VisuallyHidden>
+      <span aria-hidden>
+        {number} <Typography type={currencySize}>{currency}</Typography>
```
Comment (shmidt-i) [inline src/Atoms/Number/Number.tsx]: need to somehow `inherit` weight then

Comment (JacobBlomgren) [reply src/Atoms/Number/Number.tsx]: fixed

File: src/Molecules/CardWithTitle/__snapshots__/CardWithTitle.stories.storyshot [hunk 2]
```diff
@@ -111,22 +111,22 @@ exports[`Storyshots Molecules | CardWithTitle Integration: CardWithTitle with cu
   vertical-align: middle;
 }
 
-.c4 {
+.c6 {
```
Comment (shmidt-i) [inline src/Molecules/CardWithTitle/__snapshots__/CardWithTitle.stories.storyshot]: Have you done weird formatting ?)

File: src/Molecules/Select/__snapshots__/Select.stories.storyshot [hunk 3]
```diff
@@ -230,7 +230,6 @@ exports[`Storyshots Molecules | Select Select with overflowing placeholder 1`] =
 .c1 {
   position: relative;
   border: 1px solid #BCBCB6;
-  padding-right: 28px;
```
Comment (shmidt-i) [inline src/Molecules/Select/__snapshots__/Select.stories.storyshot]: Is this ok?

File: src/common/utils.test.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { pickAriaAttributes } from './utils';
+
+test('', () => {
```
Comment (shmidt-i) [inline src/common/utils.test.ts]: Name it! 🤓

File: src/Molecules/Select/__snapshots__/Select.stories.storyshot (hunk 3 above)
Comment (JacobBlomgren) [reply src/Molecules/Select/__snapshots__/Select.stories.storyshot]: yeah I merged in some stuff from beta

File: src/common/utils.test.ts (hunk 4 above)
Comment (JacobBlomgren) [reply src/common/utils.test.ts]: sorry 🙈

**Pull request merged**